### ISSUE TITLE
Force dates to be in the future

### ISSUE THREAD

**Issue by Hardmath123** [wanasit/chrono#215]
Hi! Thanks for a fantastic library. :-)

I was wondering if there is a way to force parsed dates to be in the future. For example, today is Tuesday, August 29, 2017. If I enter "Back to school on Monday", I want that "Monday" to refer to the Monday next week (first week of September), rather than yesterday. Is there an easy way to do this? (It should also work with months, so "School's out in July" should refer to July 2018.)

**Comment by RayBB** [comment]
You should checkout #211 

It looks like they have an option called forwarddateonly

**Comment by Hardmath123** [comment]
Aha, thanks! Sadly this useful option isn't documented in the README. :-(

Use-case is https://github.com/Hardmath123/picard, by the way.

**Comment by RayBB** [comment]
Yeah, I noticed it wasn't document. I think the author just hasn't finalized it yet.

**Comment by Hardmath123** [comment]
Yes, in particular, it doesn't quite seem to work correctly with months.
